app/views/schools/participants/_ects.html.erb

```diff
@@ -25,6 +25,22 @@
   </table>
 <% end %>
 
+<% if ect_profiles.no_qts_participants.any? %>
+  <h2 class="govuk-heading-m govuk-!-margin-bottom-0">Checking QTS</h2>
+  <p class="govuk-!-margin-bottom-3">
+    These ECTs do not have qualified teacher status (QTS) yet. They need this to be eligible for funded training.
+    We’ll keep checking their status and notify you if something changes.
+  </p>
+
+  <table class="govuk-table govuk-!-margin-bottom-9" data-test="details_ects">
+    <% if FeatureFlag.active?(:change_of_circumstances) %>
```

Comment:
Is this flag still needed @tonyheadford ?

Reply:
No, but we haven't had time to go through and remove it yet

Reply:
But we don't need it in these changes, is what you mean 👍

Reply:
I left this in because of specs that check for both. We can remove it though once we get rid of the old participant_categories stuff.